### ISSUE TITLE
Replace specified (rather than all) matches with global replacement text

### ISSUE THREAD

**Issue by allysonsouza** [johngodley/search-regex#77]
### Summary

As pointe in this topic, would be great to have the option to specify which occurrence to be replaced, rather than all. In my case I'm trying to accomplish a very common scenario: replace all first images inserted in posts, a common issue when a project was previously using a them that doesn't show the featured image in single.

### Actual Behavior

There's no option to change the global matching. (Not an regex expert, I don't know if it's the right term)

### Steps to reproduce

Try to match any regex agains first or one specif occurrence.

### Environment

Firefox 84.0.2 (64-bits) Windows 10, WordPress 5.6, Search Regex 2.4.1

**Comment by johngodley** [comment]
This is something that will require you to create a regular expression that matches the first occurrence.

**Comment by allysonsouza** [comment]
That's right @johngodley, but from what I understood in the topic mentioned, and from what I remember I have tried, currently the plugin is not supporting that regular expression (not an expert and it's some time since I opened, so I cannot explain better, but the user **neilrh1** explained in the topic).

**Comment by johngodley** [comment]
This issue is old now, and there's not enough information to do anything about. If it's still something needed then please reopen.
